lib/reek/smells/unused_private_method.rb

```diff
@@ -79,8 +79,12 @@ def unused_private_methods(ctx)
       # @return [Boolean]
       #
       def ignore_method?(ctx, method)
-        ignore_methods = value(EXCLUDE_KEY, ctx, DEFAULT_EXCLUDE_SET)
-        ignore_methods.any? { |ignore_method| method.name[ignore_method] }
+        # ignore_contexts will be e.g. ["Foo::Smelly#my_method", "..."]
+        ignore_contexts = value(EXCLUDE_KEY, ctx, DEFAULT_EXCLUDE_SET)
+        ignore_contexts.any? do |ignore_context|
+          method_from_context = ignore_context.split('#').last
```

Comment:
Aren't you ignoring the whole class name part? AFAICS, this will make Reek ignore `Smelly#foobar` even if you put `Awesome#foobar` here.